heliopy/data/spice.py

```diff
@@ -51,23 +51,57 @@ def make_doc_entry(self):
 
 spacecraft_kernels = [_Kernel('Solar Orbiter 2020', 'solo_2020',
                               'https://issues.cosmos.esa.int/solarorbiterwiki/download/attachments/7274724/solo_ANC_soc-orbit_20200207-20300902_V01.bsp'),
-                      _Kernel('Helios 1', 'helios1',
-                              ['https://naif.jpl.nasa.gov/pub/naif/HELIOS/kernels/spk/100528R_helios1_74345_81272.bsp',
-                               'https://naif.jpl.nasa.gov/pub/naif/HELIOS/kernels/spk/160707AP_helios1_81272_86074.bsp'],
+                      _Kernel('Helios 1 Reconstructed', 'helios1_rec',
+                              'https://naif.jpl.nasa.gov/pub/naif/HELIOS/kernels/spk/100528R_helios1_74345_81272.bsp',
+                              'https://naif.jpl.nasa.gov/pub/naif/HELIOS/kernels/spk/aareadme_kernel_construction.txt'),
+                      _Kernel('Helios 1 Predicted', 'helios1_pred',
+                              'https://naif.jpl.nasa.gov/pub/naif/HELIOS/kernels/spk/160707AP_helios1_81272_86074.bsp',
```

Comment:
I think these should be kept as they are, since there will never be an actual reconstructed trajectory for the predicted period.

Reply:
That is true, I think they're from different timeframes. I'll look for documentation about this

Reply:
Okay! So for Reconstructed, we have,

```The position data described above are insufficient to directly generate SPK
kernels for the following reasons:
    - low numerial precision
    - incomplete state vectors (only positions, no velocities)

However, the available position data can be used to reconstruct the
trajectories of Helios 1 and 2 by finding an orbit solution that fits the
position data well. The intention behind this procedure is that the large
amount of available positions (2487 positions for Helios 1, 1516 positions for
Helios 2) compensates their low precision.```

Reply:
And for Predicted, we have,

```The position data in helorb2.asc and the simulated time interval cover
the entire active mission time of Helios 2, which failed fataly on 
1980-03-03 due to a transmitter failure. Hence the SPK kernel
100607R_helios2_76016_80068.bsp covers the entire time span during which
Helios 2 produced science data.

The situation is different for Helios 1. The position data available in
helorb1.asc only covers the time from launch to 1981-09-30. Helios 1
failed on 1986-03-15, and so there is a considerable amount of time for
which there is no orbital information available.

In order to provide trajectory information for Helios 1 for these times,
the orbital solution found for Helios 1 as described in section 2) was used
to "predict" the orbit between 1981-09-30 and 1986-03-15.````